GoogleForCreators/web-stories-wp
# Mark Coverr videos as having no sound

## Context



## Summary

Currently, the pre-publish checklist shows a message for a video with missing captions.

Videos from Coverr inserted via the 3P Media tab don't have any sound though, so the message is very misleading in that case.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

1. Create a story. 
2. Insert Coverr video. 
3. Publish story. 
4. Ensure that coverr video does not appear in accessiblity tab of PPC. 

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7865
